### PR TITLE
tests: Add tests for image export and import

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
Add missing tests from #2779

### CODE REVIEW

File: integration/components/image/image_test.go [hunk 1]
```diff
@@ -182,6 +186,22 @@ func TestImage(t *testing.T) {
...
+		{
+			name: "export",
+			cmd:  commonImage.NewExportCmd(),
+			args: []string{testRegistryImage, exportPath},
+			expectedStdout: []string{
+				fmt.Sprintf("Successfully exported images to %s", exportPath),
+			},
+		},
+		{
+			name: "import",
```
Comment (burak-ok) [inline integration/components/image/image_test.go:205]: nit: Add comment that the order of these 2 test steps are important

File: integration/components/image/image_test.go [hunk 2]
```diff
@@ -182,6 +186,22 @@ func TestImage(t *testing.T) {
...
+			cmd:  commonImage.NewImportCmd(),
+			args: []string{exportPath},
+			expectedStdout: []string{
+				fmt.Sprintf("Successfully imported images:\n  %s", testRegistryImage),
```
Comment (burak-ok) [inline integration/components/image/image_test.go:209]: Is there an easy way to check if the imported image is the same as the one we exported?

**Review by burak-ok — APPROVED** [review]
LGTM, just nits

File: integration/components/image/image_test.go (hunk 2 above)
Comment (mauriciovasquezbernal) [reply integration/components/image/image_test.go:211]: I don't think so. We could check the image digest but with the current structure of the test it's difficult to do.

**Pull request merged**